src/main/flight/imu.c

```diff
@@ -211,9 +211,13 @@ STATIC_UNIT_TESTED void imuMahonyAHRSupdate(float dt, float gx, float gy, float
 
     // Use raw heading error (from GPS or whatever else)
     float ex = 0, ey = 0, ez = 0;
+    float ez_ef = 0.0f;
```

Comment:
Yu can simply pass very small `cogYawGain` instead

Reply:
I only moved it for logging purposes.
This whole PR isn't intended to 'go anywhere' :-)
It's only to test your code in PR #12792 and to help determine the most appropriate heuristic for the rescue when your changes are applied.  Once you are happy with #12792, it should be merged first.  We should then think about what kind of 'IMU' debug we should make, going forward, since probably I should not 'hijack' the attitude debug as I have done.  I will then make a small PR with the GPS Rescue settings that we find, on testing, work best with the improved IMU gain control code.